app/Console/Commands/ImportCsv.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use League\Csv\MapIterator;
+use League\Csv\Reader;
+
+class ImportCsv extends Command
+{
+    private const MAPPED_TABLES = [
+        'imported_products',
+        'imported_variants',
+        'imported_tracklist'
+    ];
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'import:csv {csvPath} {importTable}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Parse CSV and import data in table (you need to create table before running).';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        $importTable = $this->argument('importTable');
+
+        if (!in_array($importTable, self::MAPPED_TABLES)) {
+            $this->error('Provided tabled is not mapped. Please create table and add in MAPPED_TABLES array');
+            return self::FAILURE;
+        }
+
+        $csv = Reader::createFromPath($this->argument('csvPath'));
+        $csv->setHeaderOffset(0);
+
+        $migratedRows = 0;
+        switch ($importTable) {
+            case 'imported_products':
+                $migratedRows = $this->importProducts($csv->getRecords());
+                break;
+            case 'imported_variants':
+                $migratedRows = $this->importVariants($csv->getRecords());
+                break;
+            case 'imported_tracklist':
+                $migratedRows = $this->importTracklist($csv->getRecords());
+                break;
+        }
```

Comment:
:large_blue_circle: I would use `match()` here. It's a perfect replacement for these statements

```suggestion
        $migratedRows = match ($importedTable) {
            'imported_products' => $this->importProducts($csv->getRecords()),
            'imported_variants' => $this->importVariants($csv->getRecords()),
            'imported_tracklist' => $this->importTracklist($csv->getRecords()),
        };
```